nixos/modules/services/hardware/pcscd.nix

```diff
@@ -47,10 +44,7 @@ in
     };
   };
 
-  ###### implementation
-
   config = mkIf config.services.pcscd.enable {
-
```

Comment:
Same as what I said above: I cannot see the benefits of removing a blank line and it adds some "noise" to git blame..